app/cdap/components/GridTable/index.tsx

```diff
@@ -51,6 +54,10 @@ export default function GridTable() {
       count: '0',
     },
   ]);
+  const [toaster, setToaster] = useState<ISnackbar>({
```

Comment:
See comments above on using the snackbar with a customhook - this also isn't consistent with the code you wrote on the other component. You have a constant snackbarDefaultState that isn't being used here. Either way this should probably be a custom hook.

Reply:
Done